runtime/gc_structs/VMThreadStackSlotIterator.hpp

```diff
@@ -67,7 +67,8 @@ class GC_VMThreadStackSlotIterator
 			J9MODRON_OSLOTITERATOR *oSlotIterator,
 			bool includeStackFrameClassReferences,
 			bool trackVisibleFrameDepth,
-			bool syncWithContinuationMounting = false);
+			bool syncWithContinuationMounting = false,
```

Comment:
small naming cleanup: let's rename this to isConcurrentGC (here and anywhere else this flag is used) 

then both  isConcurrentGC and isGlobalGC represent the facts as known by caller, without assuming how the callee will use them